ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-main.yaml

```
@@ -126,6 +126,159 @@ tests:
         requests:
           cpu: 300m
           memory: 500Mi
+- as: test-e2e-upstream
```

```suggestion
- as: test-e2e-upstream-4-17
```

Let's append openshift versions to this. 

I'd like to start getting ocp 4.18 in our testing soon.